framework/src/util/Utils.cpp

```diff
@@ -154,6 +154,13 @@ std::string GetCurrentWorkingDirectory()
   return std::string();
 }
 
+static const std::string s_CurrWorkingDir = InitCurrentWorkingDirectory();
```

Comment:
By reading msdn for `SearchPath`, it seems that you have to provide a file name and it tries to find an existing path in the filesystem based on some search logic. To me, it doesn't look like it would be useful for deriving e.g. the framework directory path using this function. In this case, we do not have a file name (only a partial path) and it isn't required to exist. But I haven't used `SearchPath` on my own yet.